.ci/magician/vcr/tester.go

```diff
@@ -223,12 +223,13 @@ func (vt *Tester) Run(mode Mode, version provider.Version, testDirs []string) (*
 	logFileName := filepath.Join(vt.baseDir, "testlogs", fmt.Sprintf("%s_test.log", mode.Lower()))
```

Comment:
It seems that we're facing one issue that the build log of the replaying after recording will replace the initial replaying build log as the file is named based on the test mode and they're both named "replaying_test.log"